instructors.md

```diff
@@ -90,18 +90,548 @@ which are covered in the lesson on the shell.
 
 ## [Repeating Actions with Loops](02-loop.html)
 
+Solutions to exercises:
+
+> ## From 1 to N {.challenge}
+> Using `range`,
+> write a loop that uses `range` to print the first 3 natural numbers:
+>
+> ~~~ {.output}
+> for i in range(1,4):
+>    print i
+> 1
+> 2
+> 3
+> ~~~
+
+> ## Computing powers with loops {.challenge}
+> Write a loop that calculates the same result as `5 ** 3` using
+> multiplication (and without exponentiation).
+> ~~~ {.output}
+> result = 1
+> for i in range(0,3):
+>    result = result*5
+> print result
+> 125
+> ~~~
+
+> ## Reverse a string {.challenge}
+>
+> Write a loop that takes a string,
+> and produces a new string with the characters in reverse order.
+> ~~~ {.output}
+> newstring = ''
+> oldstring = 'Newton'
+> length_old = len(oldstring)
+> for c in range(length_old):
+>    newstring = newstring + oldstring[length_old-c-1]
+> print newstring
+> 'notweN'
+> ~~~
+
```

Comment:
Can we use something more descriptive than c as the variable in the loop to demonstrate how we would like our students to program?